extensions/notebook/src/prompts/input.ts

```diff
@@ -0,0 +1,57 @@
+// This code is originally from https://github.com/DonJayamanne/bowerVSCode
+// License: https://github.com/DonJayamanne/bowerVSCode/blob/master/LICENSE
+
+import { window, InputBoxOptions } from 'vscode';
+import Prompt from './prompt';
+import EscapeException from './escapeException';
+
+const figures = require('figures');
+
+export default class InputPrompt extends Prompt {
```

Comment:
Was this copied from one of the other extensions?

Reply:
Yeah - https://github.com/microsoft/azuredatastudio/blob/7f7052ad42d9d56fbd3321d36e13ea315846c417/extensions/mssql/src/prompts/input.ts#L10

(honestly I want to get rid of all this input prompt stuff at some point. It seems like a completely unnecessarily complex wrapper over the vscode input prompt stuff for our uses of it. But that's a later task)